docs/source/author/writing_guide.rst

```diff
@@ -325,7 +334,7 @@ Ordered list:
 2. Item 2
 3. Item 3
 
-Nested lists can be created by indenting each level by 2 spaces.
+Nested lists can be created by indenting each level by 4 spaces.
 
 .. code-block:: none
 
```

Comment:
The corollary to the above item text below should be 'indented by *4* spaces'

Likewise for the corollary in item 2

(and update the resulting text also)